src/field_theory/finite_card.lean

```diff
@@ -0,0 +1,28 @@
+/-
+Copyright (c) 2019 Casper Putz. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joey van Langen, Casper Putz
+-/
+
+import algebra.char_p data.zmod.basic linear_algebra.basis
+
+universes u
+variables {α : Type u} [discrete_field α] [fintype α]
```

Comment:
I think `α` should be an explicit argument here, as it won't generally be inferable from context for the theorems below.